alphagov/tsuru-terraform
# Revert "[#96756584] Add graphite server and DNS record"

_What_

Reverts alphagov/tsuru-terraform#70 

As part of the outcomes of this morning's after standup conversation, we're removing the Graphite server and Statsd setup, opting instead for applications that want to deploy on the platform to bring their own metrics. This PR removes the servers created and the DNS entries added to both the GCE and AWS environments.

_Who can review this PR_
This can be reviewed by any member of the core team apart from @mtekel.

_How to test this_

Checkout the branch, rerun terraform and ansible, then check that the basic services (Gandalf, API server) are accessible.
